src/main.go

```diff
@@ -109,6 +109,10 @@ type tester struct {
 	// use --disable_query_log or --enable_query_log to control it
 	enableQueryLog bool
 
+	// enable result log will output to result file or not.
+	// use --enable_result_log or --disable_result_log to control it
+	enableResultLog bool
```

Comment:
I don't quite get what does 'result log' mean? Could you provide an example? Thanks.

Reply:
An example:

The test
```
--disable_result_log
show tables;
--enable_result_log
show tables;
```

The result
```
show tables;
show tables;
Tables_in_show
```

Reply:
Ref https://github.com/mysql/mysql-server/blob/ea1efa9822d81044b726aab20c857d5e1b7e046a/client/mysqltest.cc#L557

Reply:
Got it! Thanks.